mpu6050_scripts/poll-temp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+overwrite() { 
+	echo -e "\r\033[1A\033[0K$@"; 
+	}
+
+#Enable MPU6050 chip
+sudo i2cset -y 0 0x68 0x6B 0
+
+while sleep 1; 
+do
+    #Get raw data
+    HB=$(sudo i2cget -y 0 0x68 0x41)
+    LB=$(sudo i2cget -y 0 0x68 0x42)
```

Comment:
Can You read both bytes in one process call?

Reply:
Yes, it only needs to specify what size of data we want to read.
b - means one byte.
w - means one word.
Thus,
i2cget -y 0x68 0x41 w will read both bytes